osu.Game.Tests/Rulesets/TestSceneRulesetSkinProvidingContainer.cs

```diff
@@ -43,13 +43,13 @@ public void TestEarlyAddedSkinRequester()
 
             AddStep("setup provider", () =>
             {
-                var rulesetSkinProvider = new RulesetSkinProvidingContainer(Ruleset.Value.CreateInstance(), Beatmap.Value.Beatmap, Beatmap.Value.Skin);
-
-                rulesetSkinProvider.Add(requester = new SkinRequester());
-
+                requester = new SkinRequester();
                 requester.OnLoadAsync += () => textureOnLoad = requester.GetTexture("test-image");
 
-                Child = rulesetSkinProvider;
+                Child = new RulesetSkinProvidingContainer(Ruleset.Value.CreateInstance(), Beatmap.Value.Beatmap, Beatmap.Value.Skin)
+                {
+                    requester
```

Comment:
Was in the middle of leaving review on this.................

Feeling pretty bad about this. The fact that this is a _framework container_ using C# collection initialiser syntax feels very footgunny.

Like the usual idiomatic way to write this is `Child = requester`, no?

Reply:
Ah yeah. I wanted to get this PR out of the way because it was blocking me from looking at/merging others.

I've pushed the fix straight to master: https://github.com/ppy/osu/commit/a037dbf8debe7bcecfa904d68c20875d3844236d

> What was wrong with the editorconfig suppression that was here before?

Doesn't seem necessary after all. The issue is as the OP says.